### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Enzo-adr5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,5 +29,10 @@
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
     "husky": "^8.0.3"
+  },
+  "dependencies": {
+    "prop-types": "^15.8.1",
+    "react-icons": "^4.9.0",
+    "react-simple-star-rating": "^5.1.7"
```
Comment (rocambille) [inline package.json]: these should be installed here : uninstall them

File: frontend/src/components/SignIN.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../App.scss";
+
+function SignIn() {
```
Comment (rocambille) [inline frontend/src/components/SignIN.jsx]: the file name should follow the name of the component : every letter ;)

File: frontend/src/pages/Home.jsx [hunk 3]
```diff
@@ -1,3 +1,12 @@
+import React from "react";
+import SignIn from "../components/SignIN";
+import "../App.scss";
+
 export default function Home() {
-  return <h1> "hello world depuis Home !!!" </h1>;
+  return (
+    <div className="Home">
+      <h1> "hello world depuis Home !!!" </h1>;
+      <SignIn element={SignIn} />
```
Comment (rocambille) [inline frontend/src/pages/Home.jsx]: your component has no element prop

File: .prettierrc
Comment (rocambille) [inline .prettierrc:1]: this file should not be here : remove it

**Review by rocambille — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rocambille) [reply package.json]: you still have packages to uninstall ;)

**Comment by rocambille** [comment]
conflicts to resolve

File: frontend/src/App.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.Home {
```
Comment (Justine-Merlin) [inline frontend/src/App.scss]: css class (this is the same for scss) should be write in kebab-case or in camelCase, but not in PascalCase. Rename all of your class.

File: frontend/src/App.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+/*Page Register*/
+.Inscription {
+  border: 5px solid #dec439;
+  border-radius: 1rem;
+  display: grid;
+  grid-template-columns: repeat(2, 0.25fr);
+  justify-content: center;
+  grid-gap: 1rem;
+  margin: 5rem;
+  .envoie {
```
Comment (Justine-Merlin) [inline frontend/src/App.scss]: Rename it in english.

File: frontend/src/App.scss [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    text-align: center;
+    color: #dec439;
+    padding: 10%;
+    margin-top: 6rem;
+  }
+}
+
+/*Page SignIn*/
+.Sign {
+  .bouton {
```
Comment (Justine-Merlin) [inline frontend/src/App.scss]: Rename it in english.

File: frontend/src/components/SignIn.jsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    <div className="Sign">
+      <h2>
+        Explorez l'univers des saveurs vinicoles assemblez habilement différents
+        cépages pour créer votre chef-d'œuvre gustatif, une symphonie de délices
+        sur mesure.
+      </h2>
+      <div className="bouton">
+        <Link to="/register">S'inscrire</Link>
+
+        <br />
```
Comment (Justine-Merlin) [inline frontend/src/components/SignIn.jsx]: Use css to style and remove this `<br />`

File: frontend/src/contexts/AuthContext.jsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+    return { token, setToken };
+  }, [token]);
+
+  return (
+    <AuthContext.Provider value={tokenManage}>{children}</AuthContext.Provider>
+  );
+}
+
+AuthContextProvider.propTypes = {
+  children: PropTypes.node.isRequired,
```
Comment (Justine-Merlin) [inline frontend/src/contexts/AuthContext.jsx]: `node` prop-type is used to render anything, but in your case that's a React element you want to render. So the correct type is `element` [Documentation](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html#requiring-single-child)

File: package.json [hunk 9]
```diff
@@ -29,5 +29,9 @@
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
     "husky": "^8.0.3"
+  },
+  "dependencies": {
```
Comment (Justine-Merlin) [inline package.json]: Remove these packages from here with `npm rm ...` or `npm uninstall ...`

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

**Pull request merged**